modules/mono/csharp_script.cpp

```diff
@@ -1328,16 +1328,22 @@ bool CSharpScript::_update_exports() {
 		}
 
 		GDMonoClass *top = script_class;
+		List<GDMonoClass *> classes;
 
 		while (top && top != native) {
-			const Vector<GDMonoField *> &fields = top->get_all_fields();
+			classes.push_back(top);
+			top = top->get_parent_class();
+		}
+
+		for (int i = classes.size() - 1; i >= 0; i--) {
```

Comment:
This is better as `for (List<GDMonoClass *>::Element E = classes.back(); E; E = E->prev())` since we are dealing with a linked list.

Reply:
Thanks, I'll do that instead :)